[2024-09-24T00:30:16.544+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion_15min.trigger_stage_processing scheduled__2024-09-24T00:15:00+00:00 [queued]>
[2024-09-24T00:30:16.556+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion_15min.trigger_stage_processing scheduled__2024-09-24T00:15:00+00:00 [queued]>
[2024-09-24T00:30:16.557+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-09-24T00:30:16.576+0000] {taskinstance.py:1382} INFO - Executing <Task(TriggerDagRunOperator): trigger_stage_processing> on 2024-09-24 00:15:00+00:00
[2024-09-24T00:30:16.582+0000] {standard_task_runner.py:57} INFO - Started process 2422 to run task
[2024-09-24T00:30:16.585+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion_15min', 'trigger_stage_processing', 'scheduled__2024-09-24T00:15:00+00:00', '--job-id', '553', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion_15min.py', '--cfg-path', '/tmp/tmptpnzwpbi']
[2024-09-24T00:30:16.586+0000] {standard_task_runner.py:85} INFO - Job 553: Subtask trigger_stage_processing
[2024-09-24T00:30:16.730+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion_15min.trigger_stage_processing scheduled__2024-09-24T00:15:00+00:00 [running]> on host airflow-worker
[2024-09-24T00:30:16.939+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion_15min' AIRFLOW_CTX_TASK_ID='trigger_stage_processing' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T00:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-24T00:15:00+00:00'
[2024-09-24T00:30:17.023+0000] {trigger_dagrun.py:173} INFO - Clearing stage_processing on 2024-09-24T00:00:00+00:00
[2024-09-24T00:30:17.231+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:30:27.268+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:30:37.383+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:30:47.442+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:30:57.506+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/trigger_dagrun.py", line 220, in execute
    raise AirflowException(f"{self.trigger_dag_id} failed with failed states {state}")
airflow.exceptions.AirflowException: stage_processing failed with failed states failed
[2024-09-24T00:30:57.510+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=raw_ingestion_15min, task_id=trigger_stage_processing, execution_date=20240924T001500, start_date=20240924T003016, end_date=20240924T003057
[2024-09-24T00:30:57.528+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 553 for task trigger_stage_processing (stage_processing failed with failed states failed; 2422)
[2024-09-24T00:30:57.542+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-24T00:30:57.594+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
